src/radiotray-ng/gui/appindicator/po/en.po

```diff
@@ -0,0 +1,152 @@
+# English translations for PACKAGE package.
+# Copyright (C) 2019 radiotray-ng team
+# This file is distributed under the same license as the PACKAGE package.
+# Automatically generated, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: https://github.com/ebruck/radiotray-ng/issues\n"
+"POT-Creation-Date: 2019-07-30 01:11+0300\n"
+"PO-Revision-Date: 2019-07-23 23:51+0300\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: ../../../radiotray_ng.cpp:595
+msgid "(Mute)"
+msgstr ""
+
+#: ../appindicator_gui.cpp:483
+msgid "About"
+msgstr "About"
+
+#: ../appindicator_gui.cpp:271
+msgid "Bitrate"
+msgstr "Bitrate"
+
+#: ../appindicator_gui.cpp:456
+msgid "Bookmark Editor..."
+msgstr "Bookmark Editor..."
+
+#: ../../../radiotray_ng.cpp:656
+msgid "Bookmarks Reload Failed"
+msgstr ""
```

Comment:
There are several of these that are blank.

Reply:
Its ok for en - gettext will use original string.

Reply:
fixed